linux/src/threadpool_linux.c

```diff
@@ -312,28 +246,17 @@ static int reallocate_threadpool_array(THREADPOOL* threadpool)
                 }
                 threadpool->task_array = temp_array;
 
-                // Ensure there are no gaps in the array
-                // Consume = 2
-                // Produce = 2
-                // [x x x x]
-                // Consume = 3
-                // Produce = 2
-                // [x x 0 x]
-                // During resize we will have to memmove in the gap at 2
-                // [x x 0 x 0 0 0 0]
-
                 int64_t insert_pos = interlocked_add_64(&threadpool->insert_idx, 0) % existing_count;
                 int64_t consume_pos = interlocked_add_64(&threadpool->consume_idx, 0) % existing_count;
-
                 uint32_t compress_count = 0;
```

Comment:
>compress [](http://example.com/codeflow?start=25&length=8)

It's not really compress as in compression (deflate, zip, etc.)  Likely a better name would be move_count

Reply:
deleted